### ISSUE TITLE
Some Patter chat channels aren't showing up

### ISSUE THREAD

**Issue by bazbt3** [bazbt3/PigPen#30]
Specifically:
* ThemeMonday
* CryptoNotifier
* Community Wiki

It's the channels with added avatars.

**Comment by bazbt3** [comment]
This update works in `getchannel`, however `getsubscribed` seems more complex than it needs to be.

```
try:
		channelname = channelcontent[0]["raw"][0]["value"]["name"]
	except:
		channelname = channelcontent[0]["raw"][1]["value"]["name"]
	else:
		channelname = "PM"
```

**Comment by bazbt3** [comment]
Fixed in v0.3.7.

**Comment by bazbt3** [comment]
Fixed in v0.3.8.

**Comment by bazbt3** [comment]
Incidental: Cryptonotifier isn't visible because it's a private channel. I should add the ability to unsubscribe from channels next.
